src/main/java/frc/robot/subsystems/Drivetrain/Drive.java

```diff
@@ -12,5 +15,8 @@
     double getHeading();
     void resetEncoderTicks();
     Pair<Double, Double> distanceDriven();
+
+    CANSparkMax getController(int side);
+    RelativeEncoder getEncoder(int side);
```

Comment:
Do we really need to return the whole `RelativeEncoder` object reference?

I think we could just have a simple `Pair<Double, Double> getEncoderTicks()` method that returns a `Pair.of()` the two values. This keeps the concerns better separated and doesn't expose anything more than is really required.

Reply:
just realized we don't, my math can also be optimized since ticksPer cancels out if revs = encoderTicks / ticksPer